packages/zowe-explorer-api/src/vscode/ui/PaginationCodeLens.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as vscode from "vscode";
+
+export class LoadMoreCodeLens implements vscode.CodeLensProvider {
+    public constructor(private commandId: string) {}
+    public provideCodeLenses(document: vscode.TextDocument, token: vscode.CancellationToken): vscode.ProviderResult<vscode.CodeLens[]> {
```

Comment:
In addition to the previous comment, we can rename this function to `loadCodeLenses` or something similar 😋